packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.scss

```diff
@@ -128,6 +128,65 @@ $select-menu-max-width: 300px;
     }
   }
 
+  &__help_popover {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    .select-menu__summary.amp-button.amp-button--text {
+      color: var(--black80);
+      &.highlight,
+      &:hover,
+      &:focus,
+      &:active {
+        color: var(--black100);
+      }
+    }
+
```

Comment:
@EugeneTseitlin can you please have a look at this?
Do we need all the CSS locally? I expect the styles to be inherited from the component directly from the design system

Reply:
@EugeneTseitlin ?